docs/_sidebar.md

```diff
@@ -1,31 +1,34 @@
 - **Getting Started**
   - [Introduction](index 'Contribute to the freeCodeCamp.org Community')
-  - [Frequently Asked Questions](/FAQ.md)
+  - [Frequently Asked Questions](FAQ.md)
 - **Code Contribution**
-- - [Set up freeCodeCamp locally](/how-to-setup-freecodecamp-locally.md)
-  - [Open a pull request](/how-to-open-a-pull-request.md)
-  - [Work on coding challenges](/how-to-work-on-coding-challenges.md)
-  - [Work on video challenges](/how-to-help-with-video-challenges.md)
-  - [Work on the news theme](/how-to-work-on-the-news-theme.md)
-  - [Work on the docs theme](/how-to-work-on-the-docs-theme.md)
+- - [Set up freeCodeCamp locally](how-to-setup-freecodecamp-locally.md)
+  - [Open a pull request](how-to-open-a-pull-request.md)
+  - [Work on coding challenges](how-to-work-on-coding-challenges.md)
+  - [Work on video challenges](how-to-help-with-video-challenges.md)
+  - [Work on the news theme](how-to-work-on-the-news-theme.md)
+  - [Work on the docs theme](how-to-work-on-the-docs-theme.md)
 - **Optional Guides**
-  - [Catch outgoing emails locally](/how-to-catch-outgoing-emails-locally.md)
-  - [Set up freeCodeCamp on WSL](/how-to-setup-wsl.md)
+  - [Catch outgoing emails locally](how-to-catch-outgoing-emails-locally.md)
+  - [Set up freeCodeCamp on WSL](how-to-setup-wsl.md)
 
 ---
 
+<!-- The section below should not use relative linking -->
+
 - **中文社区贡献指南**
-  - [成为专栏作者](/i18n/chinese/news-author-application.md)
-  - [文章翻译计划](/i18n/chinese/news-translations.md)
-  - [视频翻译计划](/i18n/chinese/video-translations.md)
+  - [成为专栏作者](/i18n/Chinese/news-author-application.md)
+  - [文章翻译计划](/i18n/Chinese/news-translations.md)
+  - [视频翻译计划](/i18n/Chinese/video-translations.md)
```

Comment:
This is now the only section with `Chinese` instead of `chinese`

I assume all the rest should be capital `C`

Reply:
Yeah, that should be fixed when we sync-up with Crowdin.